server/playground.py

```diff
@@ -359,7 +359,7 @@ async def handle_mongo_translated_backend_request():
             client_session_keep_alive=True,
         )
     except Exception:
-        _SNOWFLAKE_CONNECTION = None
+        pass
```

Comment:
Maybe we can just log that the connection failed?

Reply:
Sure! Is a simple `print` okay for you ?

Reply:
just to make sure it's not silent in the console

Reply:
Added a print and checked, it's visible in the console :slightly_smiling_face: